libs/util/nestjs/kafka/src/producer/KafkaProducer.module.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  KafkaMessageJsonSerializer,
+  KAFKA_SERIALIZER,
+} from "@amplication/util/kafka";
+import { Module } from "@nestjs/common";
+import { ClientsModule } from "@nestjs/microservices";
+import {
+  createNestjsKafkaConfig,
+  KAFKA_CLIENT,
+} from "../createNestjsKafkaConfig";
+import { KafkaProducerService } from "./KafkaProducer.service";
+
+@Module({
+  imports: [
+    ClientsModule.registerAsync([
+      {
+        name: KAFKA_CLIENT,
+        useFactory: createNestjsKafkaConfig,
+      },
+    ]),
+  ],
+  providers: [
+    {
+      provide: KAFKA_SERIALIZER,
+      useClass: KafkaMessageJsonSerializer,
+    },
+    KafkaProducerService,
+  ],
+  exports: [KAFKA_SERIALIZER, KafkaProducerService],
+})
+export class KafkaProducerModule {}
```

Comment:
I think that this Module actually enable Consumer also. I suggest to make it KafkaModule.

Reply:
It makes perfect sense to me. Updated